configuration/globalproperties/clinical_gp.xml

```diff
@@ -70,46 +70,130 @@
     v.date_stopped is null
     order by en.encounter_datetime desc</value>
     </globalProperty>
     <globalProperty>
-      <property>emrapi.sqlSearch.queuePatients</property>
-      <value>select distinct concat(pn.given_name," ", ifnull(pn.family_name,'')) as name,
-  pi.identifier as identifier,
-  concat("",p.uuid) as uuid,
-  concat("",v.uuid) as activeVisitUuid,
-  IF(va.value_reference = "Admitted", "true", "false") as hasBeenAdmitted
-  from
-    visit v join person_name pn on v.patient_id = pn.person_id and pn.voided = 0 and v.voided=0
-    join patient_identifier pi on v.patient_id = pi.patient_id and pi.voided=0
-    join patient_identifier_type pit on pi.identifier_type = pit.patient_identifier_type_id
-    join global_property gp on gp.property="bahmni.primaryIdentifierType" and gp.property_value=pit.uuid
-    join person p on p.person_id = v.patient_id  and p.voided=0
-    join encounter en on en.visit_id = v.visit_id and en.voided=0
-    join encounter_provider ep on ep.encounter_id = en.encounter_id  and ep.voided=0
-    join provider pr on ep.provider_id=pr.provider_id and pr.retired=0
-    join person per on pr.person_id=per.person_id and per.voided=0
-    join location l on l.uuid=${visit_location_uuid} and l.location_id = v.location_id
-    join obs ob on ob.encounter_id = en.encounter_id and ob.voided=0 and ob.value_coded = (
-        select distinct concept_id from concept_name where name = "Haiti_Queue"
-    )
-    join ( select obs_id from obs where voided=0 and obs_datetime in (select max(obs_datetime) from obs group by person_id, concept_id) ) ob2 on ob.obs_id = ob2.obs_id
-    left outer join visit_attribute va on va.visit_id = v.visit_id and va.voided = 0 and va.attribute_type_id = (
-				select visit_attribute_type_id from visit_attribute_type where name="Admission Status"
-			)
-
-    where
-    v.date_stopped is null
-    order by en.encounter_datetime desc</value>
+      <property>emrapi.sqlSearch.prenatalPatients</property>
+      <value>select distinct
+        concat(pn.given_name,' ', ifnull(pn.family_name,'')) as name,
+        pi.identifier as identifier,
+        concat("",p.uuid) as uuid,
+        concat("",v.uuid) as activeVisitUuid,
+        IF(va.value_reference = "Admitted", "true", "false") as hasBeenAdmitted
+      from visit v
+      join person_name pn on v.patient_id = pn.person_id and pn.voided = 0
+      join patient_identifier pi on v.patient_id = pi.patient_id
+      join patient_identifier_type pit on pi.identifier_type = pit.patient_identifier_type_id
+      join global_property gp on gp.property="bahmni.primaryIdentifierType" and gp.property_value=pit.uuid
+      join person p on p.person_id = v.patient_id
+      join location l on l.uuid = ${visit_location_uuid} and v.location_id = l.location_id
+      join visit_type vt on vt.visit_type_id = v.visit_type_id and vt.retired = 0 and vt.uuid = (
+        select distinct uuid from visit_type where name = "Prenatal" and retired = 0
```

Comment:
Let's use `uuid` for the Visit Types as well, even though we have less control over their UUID. MDS packages do not always enforce a specific UUID (I believe). But in our use case, where we create metadata from blank db, it should do.
The up side will be that we can rename the Visit Types without editing these queries.